docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1099 @@
+# Voice Control Manager
+Voice control manager features allow you to record voice and give responses for the recognized voice commands. You can register general and system voice commands such as "power on", "power off", "music play", "music stop", and so on. In addition, you can start and stop voice recording. When the voice recording is finished, you can receive multiple recognition results such as Automatic Speech Recognition (ASR) and matched commands from the commands list, which is registered by the application using the voice control client.
+
+
+The main features of the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) namespace include:
+
+-   Creating a handle and registering event handlers.
+    -   You can create a voice control manager handle and only one voice control manager instance can work on the device.
+    -   You can [get notifications](#callback) of state changes, language changes, recognition results, and errors by registered event handlers.
+-   Managing commands
+    -    You can register commands as System, Widget, foreground, SystemBackground, and background type on the voice control service. When you speak a registered command, the callback returns the recognized result.
```

Comment:
Question. In the below content, the priorities are using capital letters for the first letter.

At here, all are in small letters. Is it ok to write in this way?

Reply:
@surbhig12 Done.